### PR TITLE
ECL global keyring section

### LINKED ISSUE #99: Document ECL changes with introduction of global keyring

### PR DESCRIPTION
**Pull request by cclerget**
Fixes #99

### CODE REVIEW

File: configfiles.rst [hunk 1]
```diff
@@ -592,6 +592,24 @@ are allowed to run.
     To temporarily permit the use of legacy insecure signatures, set
     ``legacyinsecure = true`` in ``ecl.toml``.
 
+Manage ECL public keys
```
Comment (dctrud) [inline configfiles.rst:595]:
```suggestion
Managing ECL public keys
```

File: configfiles.rst [hunk 2]
```diff
@@ -592,6 +592,24 @@ are allowed to run.
...
+======================
+
+Since Singularity 3.6.0, public keys associated to ECL key fingerprints
```
Comment (dctrud) [inline configfiles.rst:598]:
```suggestion
In Singularity 3.6, public keys associated with fingerprints specified in ECL rules
```

File: configfiles.rst [hunk 3]
```diff
@@ -592,6 +592,24 @@ are allowed to run.
...
+are required to be present in user's local keyring which is not very
```
Comment (dctrud) [inline configfiles.rst:599]:
```suggestion
were required to be present in user's local keyring which is not very
```

File: configfiles.rst [hunk 4]
```diff
@@ -592,6 +592,24 @@ are allowed to run.
...
+``--global`` option was added for:
+
+  * ``singularity key import`` (root user only)
+  * ``singularity key pull`` (root user only)
+  * ``singularity key remove`` (root user only)
+  * ``singularity key export``
+  * ``singularity key list``
+
+.. note::
+    For obvious security reasons, it is not possible to import private keys
```
Comment (dctrud) [inline configfiles.rst:611]: It won't be obvious to some people who are not really aware of the nature of asymmetric keys. Would be better to remove "obvious" or give a specific reason.

**Review by dctrud — CHANGES_REQUESTED** [review]
Some comments left. Also the section should probably mention the filename of the global keyring, as all other config files are mentioned by filename in this section.

File: configfiles.rst (hunk 4 above)
Comment (cclerget) [reply configfiles.rst]: fixed

File: configfiles.rst [hunk 5]
```diff
@@ -592,6 +592,24 @@ are allowed to run.
     To temporarily permit the use of legacy insecure signatures, set
     ``legacyinsecure = true`` in ``ecl.toml``.
 
+Managing ECL public keys
+======================
```
Comment (dctrud) [inline configfiles.rst:596]:
```suggestion
=======================
```

This is what's causing CI to fail.

Comment (cclerget) [reply configfiles.rst]: fixed by https://github.com/hpcng/singularity-admindocs/pull/107/commits/6c8aafa586e56bc2a35dc65c5b05ab453ebf549c

**Review by dctrud — APPROVED** [review]

**Pull request merged**